runtime/westend/Cargo.toml

```diff
@@ -113,6 +118,7 @@ std = [
 	"pallet-authority-discovery/std",
 	"pallet-authorship/std",
 	"pallet-balances/std",
+	"pallet-beefy/std",
```

Comment:
Should we add `beefy-primitives/std` as well?

Reply:
Done, below

Reply:
Done